src/prefect/settings.py

```diff
@@ -1292,6 +1292,31 @@ def default_cloud_ui_url(settings, value):
 Maximum number of processes a runner will execute in parallel.
 """
 
+PREFECT_RUNNER_POLL_FREQUENCY = Setting(int, default=10)
+"""
+Number of seconds a runner should wait between queries for scheduled work.
+"""
+
+PREFECT_RUNNER_HEALTH_CHECK_DELAY = Setting(int, default=20)
```

Comment:
I think this setting should be a multiplier that is applied to the runner's poll frequency. That way we can say, "a runner is unhealthy after x missed polls" and prevent any dissonance by raising the poll frequency, but not adjusting this setting.

Reply:
ah gotcha, that makes sense

Reply:
updated